CMakeLists.txt

```diff
@@ -18,8 +18,9 @@ endif()
 # s2geometry needs to use the same C++ standard that absl used to avoid
 # undefined symbol errors since ABSL_HAVE_STD_STRING_VIEW etc will
 # end up defined differently.  There is probably a better way to achieve
-# this than assuming what absl used.
-set(CMAKE_CXX_STANDARD 11)
+# this than assuming what absl used. 
+# Using CACHE allows the user to override the default.
+set(CMAKE_CXX_STANDARD 11 CACHE STRING "The C++ standard to build with")
```

Comment:
https://github.com/google/s2geometry/issues/270#issuecomment-1235346863 suggests `option`. What's the difference between `set(CACHE)` and `option` / when would I want to use one or the other?  @jievince

Reply:
Hi @jmr, I am unfamiliar with CMake, and I don't know the difference between them. I just tried `option`, and it works.

Reply:
Hi @LafeWessel, can you explain it?